site/docs/concepts/config-policies.md

```diff
@@ -214,3 +214,56 @@ config[{"capability": "copy", "decision": decision}] {
 }
 ```
 Note that an empty ConfigDecisions map will be returned if the expiration date is exceeded by the time when the policy is applied. 
+
+## Taking infrastructure metrics into consideration
+
+When writing configuration policies, infrastructure metrics and costs may also be taken into account in order to optimize the generated data plane. 
+For example, selection of a storage account may be based on a storage cost, selection of a cluster may provide a restriction on cluster capacity, and so on. 
+Infrastructure attributes and metrics are stored in `/tmp/adminconfig/infrastructure.json` directory of the manager pod. Collection of the metrics and their dynamic update is beyond the scope of Fybrik.
```

Comment:
stored in **the** ....
And, add a sentence that says... One may develop or use 3rd party solutions for monitoring and updating these infrastructure metrics.

Reply:
Done.